forte/integrals/cholesky_integrals.h

```diff
@@ -80,6 +80,14 @@ class CholeskyIntegrals : public Psi4Integrals {
     void set_tei(size_t p, size_t q, size_t r, size_t s, double value, bool alpha1,
                  bool alpha2) override;
 
+    void build_from_asints(std::shared_ptr<ActiveSpaceIntegrals> as_ints) {}
```

Comment:
I wonder if these function should actually be implemented in `integrals.h` and throw an error saying not implemented or should be overloaded to avoid this type of code in three-indexed integrals.

Reply:
Agree with the comment above. These should be implemented in the base class and throw. Classes that implement the functionality should override the base class implementation. Please fix.